src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -37,8 +37,14 @@ public Monument getMonument(@PathVariable("id") Integer id) throws ResourceNotFo
     }
 
     @PutMapping("/api/monument/{id}")
-    public Monument updateMonument(@PathVariable("id") Integer id, Monument monument) {
-        this.monumentService.update(monument);
+    public Monument updateMonument(@PathVariable("id") Integer id, @RequestBody Monument monument) {
+        monument.setId(id);
+        System.out.println("in put " + monument.toString());
+        try {
+            this.monumentService.update(monument);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Debug print still left?

Reply:
Intentional

Reply:
oh wait line 42. never mind I'll remove that